core/src/filter/resources/hudson/model/hudson-version.properties

```diff
@@ -1 +1,2 @@
+# TODO unused?
```

Comment:
Last time I checked that, there was some tooling using it. I do not remember what exactly, and it does not matter much at the moment